src/Bootstrappers/PrefixCacheTenancyBootstrapper.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Bootstrappers;
+
+use Illuminate\Cache\Repository;
+use Illuminate\Support\Facades\Cache;
+use Stancl\Tenancy\Contracts\TenancyBootstrapper;
+use Stancl\Tenancy\Contracts\Tenant;
+
+class PrefixCacheTenancyBootstrapper implements TenancyBootstrapper
+{
+    protected null|string $originalPrefix = null;
+    protected string $storeName;
+
+    public function bootstrap(Tenant $tenant): void
+    {
+        $this->originalPrefix = config('cache.prefix');
+        $this->storeName = config('cache.default');
+
+        $this->setCachePrefix('tenant_id_' . $tenant->id);
```

Comment:
The prefix should be customizable in the config. Take a look at the config file and see how all of the other prefixes we use are handled 👍🏻